### ISSUE TITLE
[Bug] Holograms can imitate power orbs in the power orb display

### ISSUE THREAD

**Issue by ILikePlayingGames** [BiscuitDevelopment/SkyblockAddons#505]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
When a hologram is placed with a power orb name and time remaining to imitate a power orb, the hologram will be shown in the power orb display.

**To Reproduce**
Steps to reproduce the behavior (how did you get this bug to occur):
1. Name a hologram with the name of a power orb and its time remaining, like "Overflux 420s"
2. Place the hologram
3. The hologram shows up on the power orb display

**Expected behavior**
The hologram shouldn't show up on the display.

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5f0e6ddd5c24786a640d19c2/ab9fd0ba-bf08-4957-8e47-81e0c6afb6a8)

**Crash Report**
If your game crashed, please paste the beginning of your crash report here. Copy from the beginning until the line shown below.
> A detailed walkthrough of the error, its code path and all known details is as follows:

**System Information**
 - OS: Windows 10
 - Mod Loader: Forge
 - Minecraft Version/Mod Loader Version: e.g. 1.8.9 - 11.15.1.2318
 - SkyblockAddons Version: 1.6.0-b.9

**Other Installed Mods**
If you have any other mods installed, please list them here along with their version number.
- Smooth Font 1.8.9-1.15.2
- Autotip 3.0
- Replaymod 1.8.9-2.5.1
- ModCore 0.1.47
- NEU 1.7.3-REL
- Patcher 41

**Additional info**
Armor stand NBT:
```text
Class: EntityArmorStand
Name: §5Overflux §e420s
NBT Data:
{
    NoGravity: 0b,
    HurtByTimestamp: 0,
    Attributes: [{
        Base: 20.0d,
        Name: "generic.maxHealth"
    }, {
        Base: 0.0d,
        Name: "generic.knockbackResistance"
    }, {
        Base: 0.10000000149011612d,
        Name: "generic.movementSpeed"
    }],
    Invulnerable: 0b,
    ShowArms: 0b,
    PortalCooldown: 0,
    AbsorptionAmount: 0.0f,
    FallDistance: 0.0f,
    DisabledSlots: 0,
    DeathTime: 0s,
    Pose: {
    },
    HealF: 20.0f,
    Invisible: 1b,
    Motion: [0.0d, 0.0d, 0.0d],
    Small: 0b,
    UUIDLeast: -4760465450654527692L,
    Health: 20s,
    Silent: 1b,
    Air: 300s,
    OnGround: 0b,
    Dimension: 0,
    Rotation: [0.0f, 0.0f],
    UUIDMost: -6234059879465398301L,
    CustomName: "§5Overflux §e420s",
    Equipment: [{
    }, {
    }, {
    }, {
    }, {
    }],
    Pos: [5.75d, 95.5d, 22.53125d],
    Fire: 0s,
    NoBasePlate: 0b,
    HurtTime: 0s,
    CustomNameVisible: 1b
}
```

**Comment by Phoube** [comment]
Lol of course Hypixel would add this to the game. Now we gotta check whether there's another armorstand near it that matches the texture url/skull id of the orb itself

**Comment by ILikePlayingGames** [comment]
Yes, that would be the most accurate way. I wonder if we can do it quicker by just checking if the armor stand is wearing a skull since holograms have no equipment.
